### ISSUE TITLE
blank node after parsing

### ISSUE THREAD

**Issue by TamouzeAssi** [RDFLib/rdflib#857]
I want to get the triple with a given blank node. In rdf file we have a string rfor a given blank node, but after  parsing the rdf file by rdflib the name for the correponding blank node is different. So how can keep track of the orginal blank in the rdf file

**Comment by TomConlin** [comment]
Attach a meaningful label, 
more or less by definition, you can not rely on __blank__ node identifier to remain consistent beyond the process that generates them. 
(In this case, the process that wrote the triple is only responsible for bnodes in that file) 
to ensure uniqueness the next process (where you are reading into rdflib) 
must be free to rewrite those identifiers to avoid collisions, even if there are not any.

**Comment by TamouzeAssi** [comment]
@TomConlin  Sorry but your answer is not clear for me. As as result, can i preserve the same node id or not please?

**Comment by TomConlin** [comment]
No,  every tool must be free to re write blank node identifiers.

https://www.w3.org/TR/rdf11-concepts/#section-blank-nodes
